docs/dev/nightly_data_builds.rst

```diff
@@ -14,12 +14,29 @@ pushes a Docker image with PUDL installed to `Docker Hub <https://hub.docker.com
 and deploys the image as a container to a GCE instance. The container runs the ETL and
 tests, then copies the outputs to a public AWS S3 bucket for distribution.
 
+
 Breaking the Builds
 -------------------
 The nightly data builds based on the ``main`` branch are our comprehensive integration
-tests. When they pass, we consider the results fit for public consumption.  The builds
-are expected to pass. If they don't then someone needs to take responsibility for
-getting them working again with some urgency.
+tests. When they pass, we consider the results fit for public consumption.
+
+When they don't pass, we need to fix them.
+
+Every morning, someone from inframundo will check the #pudl-deployments slack
+channel to see if the build has succeeded.
+
+If it has succeeded, they will track it in `this spreadsheet <https://docs.google.com/spreadsheets/d/11YdknGi4br51kxz03nNmxD-lOvilWFgm3jkUpiEdAzU/edit#gid=1678819446>`__.
+
+If it has failed, they will:
+
+1. Create a GitHub issue for the nightly build failure to hold the investigation & discussion.
+2. Track the GitHub issue and the build status in the above spreadsheet.
+3. Look in the logs and determine whether it was an "infrastructure failure," i.e. something went wrong with the code that *runs*
+the nightly build, or a "PUDL failure," i.e. something that went wrong with the PUDL ETL itself.
+4. Investigate the source of the issue & explore ways to fix it. Get help from the folks whose PRs broke the build.
```

Comment:
Do you think we need more guidance here? TBH I sort of ran out of doc-writing steam for this afternoon. But I also think this is probably good enough?